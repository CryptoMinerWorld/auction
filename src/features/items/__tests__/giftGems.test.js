import { testStateMachine } from 'react-automata';
// import { TestGiftGems } from '../components/GiftGems';

// jest.mock('firebase');

const fixtures = {
  initialData: {
    gemsContract: '0x',
    to: '123',
    from: '456',
    tokenId: '789',
  },
};

test.skip('GiftGems works', () => {
  testStateMachine(TestGiftGems, { fixtures });
});
