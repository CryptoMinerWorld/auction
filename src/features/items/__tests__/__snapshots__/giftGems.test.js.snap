// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GiftGems works: confirm 1`] = `
<div>
  <h1>
    hello
  </h1>
</div>
`;

exports[`GiftGems works: done 1`] = `
<div>
  <h1>
    hello
  </h1>
</div>
`;

exports[`GiftGems works: error 1`] = `
<div>
  <h1>
    hello
  </h1>
</div>
`;

exports[`GiftGems works: idle 1`] = `
<div>
  <h1>
    hello
  </h1>
</div>
`;

exports[`GiftGems works: loading 1`] = `
<div>
  <h1>
    hello
  </h1>
</div>
`;

exports[`GiftGems works: transferring 1`] = `
<div>
  <h1>
    hello
  </h1>
</div>
`;
